### ISSUE TITLE
Accessibility: Built-in VSCode webviews fails for Name, Role, Value (Level A) SC 4.1.2 (WCAG 2.2)

### ISSUE THREAD

**Issue by RushikeshTrivedi** [microsoft/vscode#321616]
Does this issue occur when all extensions are disabled?: Yes

VS Code Version: 1.119.0
OS Version: Windows 11

**WCAG Requirement:**

https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

**Steps to Reproduce:**

1. Open VSCODE
2. Select Extension 
3. Run accessibility checker on it

**Description:**
When we run accessiblity checker on any built in VSCODE webviews, it fails for "The ARIA role 'document' is not valid for the element <body>"

Below is the example for built-in webview in VSCODE:

<img width="2129" height="2142" alt="Image" src="https://github.com/user-attachments/assets/07909503-caac-4750-8949-6e028883209f" />

<img width="2402" height="2210" alt="Image" src="https://github.com/user-attachments/assets/a3adb697-c4bc-4f26-8eb2-01418b13d375" />

**Rationale:**
The intent of this success criterion is to ensure that Assistive Technologies (AT) can gather appropriate information about, activate (or set) and keep up to date on the status of user interface controls in the content.

When standard controls from accessible technologies are used, this process is straightforward. If the user interface elements are used according to specification the conditions of this provision will be met. (See examples of Success Criterion 4.1.2 Name, Role, Value below)

If custom controls are created, however, or interface elements are programmed (in code or script) to have a different role and/or function than usual, then additional measures need to be taken to ensure that the controls provide important and appropriate information to assistive technologies and allow themselves to be controlled by assistive technologies.

What roles and states are appropriate to convey to assistive technology will depend on what the control represents. Specifics about such information are defined by other specifications, such as [WAI-ARIA](https://www.w3.org/TR/wai-aria/), or the relevant platform standards. Another factor to consider is whether there is sufficient [accessibility support](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value#dfn-accessibility-supported) with assistive technologies to convey the information as specified.

A particularly important state of a user interface control is whether or not it has focus. The focus state of a control can be programmatically determined, and notifications about change of focus are sent to user agents and assistive technology. Other examples of user interface control states are whether or not a checkbox or radio button has been selected, or whether a collapsible tree view or accordion is expanded or collapsed.
